pkgs/development/libraries/criterion/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit, cmake, pkg-config, boxfort, libcsptr, dyncall, nanomsg }:
+
+stdenv.mkDerivation rec {
+  pname = "criterion";
+  version = "2.3.3";
+
+  src = fetchgit
```

Comment:
No `fetchFromGitHub`?

Reply:
I used `fetchgit` for the hash, so I thought `fetchFromGitHub` would come out different, because of the recursive fetch. Will update.

Reply:
Fixed.